src/ol_orchestrate/assets/openedx_course_archives.py

```diff
@@ -0,0 +1,54 @@
+import hashlib
+import json
+from pathlib import Path
+
+from dagster import (
+    AssetExecutionContext,
+    AssetIn,
+    AssetKey,
+    AutoMaterializePolicy,
+    DataVersion,
+    Output,
+    asset,
+)
+
+from ol_orchestrate.assets.edxorg_archive import course_and_source_partitions
+from ol_orchestrate.lib.openedx import process_course_xml
+
+
+@asset(
+    key=AssetKey(("edxorg", "raw_data", "course_xml")),
+    partitions_def=course_and_source_partitions,
+    group_name="edxorg",
+)
+def dummy_edxorg_course_xml(): ...
+
+
+@asset(
+    key=AssetKey(("edxorg", "raw_data", "course_metadata")),
```

Comment:
```suggestion
    key=AssetKey(("edxorg", "processed_data", "course_metadata")),
```
I think this makes sense to be under the `processed_data` hierarchy since it's not strictly raw information

Reply:
That also matches with the key path that is being constructed in the function body